lib/src/center.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:turf/helpers.dart';
+
+Position center(BBox bBox) {
```

Comment:
this is very different from the turf.js implementation:
https://github.com/Turfjs/turf/blob/master/packages/turf-center/index.ts

They have a `GeoJSONObject` as parameter and run the `bbox(...)` function on it.
I would prefer this function to be compatible with the JS-one.